config/validate_test.go

```diff
@@ -76,6 +76,7 @@ func Test_validateType(t *testing.T) {
 		{"url blank", "k", "", "url", `field k may not be blank`},
 		{"url noscheme", "k", "example.com", "url", `field k (example.com) must be a valid URL with a host`},
 		{"url badscheme", "k", "ftp://example.com", "url", `field k (ftp://example.com) must use an http or https scheme`},
+		{"memorysize", "k", "test", "memorysize", `field k (test) must be a valid memory size like '1Gb' or '100_000_000'`},
```

Comment:
I think this isn't enough. You've used RamInBytes, and I had used FromHumanSize, and I think we need *both* (or our own code) to be able to handle all of the cases like Gb, GiB, etc.

Reply:
I think using `FromHumanSize` was incorrect as the actual Unmarshalling has always used `RamInBytes`. The validator should validate based on what we place to use in the unmarshalling.  

I will add some positive test scenarios that confirms `G`, `Gi`, and `GiB` pass the validation.